kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -99,25 +112,17 @@ Exercises.PerseusBridge = {
     render_item: function(item_data) {
 
         Exercises.PerseusBridge.itemMountNode = document.createElement("div");
-        var itemRenderer = Exercises.PerseusBridge.itemRenderer = Perseus.ItemRenderer({
+
+        var ItemRenderer = React.createFactory(Perseus.ItemRenderer);
+        Exercises.PerseusBridge.itemRenderer = zk = React.render(ItemRenderer({
             item: item_data,
             problemNum: Math.floor(Math.random() * 50) + 1,
```

Comment:
This wasn't changed here, but is worth assessing; was this just a dummy problem number selector we did at the hackathon? We may want to have it be more nuanced, to avoid some of the repetition/predictability I was seeing on some exercises.